Lesson5/src/com/home/work/InputInfoObject.java

```diff
@@ -0,0 +1,99 @@
+
+public class InputInfoObject {
+
+    Passenger passenger;
+    Truck truck;
+    Militaryair militaryair;
+    Civilair civilair;
+
+    public Passenger createPessenger() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter power \t");
+        int power = scanner.nextInt();
+        System.out.print("Enter top speed \t");
+        int top_speed = scanner.nextInt();
+        System.out.print("Enter mass \t");
+        int mass = scanner.nextInt();
+        System.out.print("Enter brand \t");
+        String brand = scanner.next();
+        System.out.print("Enter count wheel \t");
+        int countWheel = scanner.nextInt();
+        System.out.print("Enter fuel consumption \t");
+        int fuelConsumption = scanner.nextInt();
+        System.out.print("Enter type of body \t");
+        String typeOfBody = scanner.next();
+        System.out.print("Enter count passengers \t");
+        int countPassengers = scanner.nextInt();
+        return passenger = new Passenger(power, top_speed, mass, brand, countWheel, fuelConsumption, typeOfBody, countPassengers);
+    }
+
+    public Truck createTruck() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter power \t");
+        int power = scanner.nextInt();
+        System.out.print("Enter top speed \t");
+        int top_speed = scanner.nextInt();
+        System.out.print("Enter mass \t");
+        int mass = scanner.nextInt();
+        System.out.print("Enter brand \t");
+        String brand = scanner.next();
+        System.out.print("Enter count wheel \t");
+        int countWheel = scanner.nextInt();
+        System.out.print("Enter fuel consumption \t");
+        int fuelConsumption = scanner.nextInt();
+        System.out.print("Enter load capacity \t");
+        int loadCapacity = scanner.nextInt();
+        return truck = new Truck(power, top_speed, mass, brand, countWheel, fuelConsumption, loadCapacity);
+    }
+
+    public Civilair createCivilair() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter power \t");
+        int power = scanner.nextInt();
+        System.out.print("Enter top speed \t");
+        int top_speed = scanner.nextInt();
+        System.out.print("Enter mass \t");
+        int mass = scanner.nextInt();
+        System.out.print("Enter brand \t");
+        String brand = scanner.next();
+        System.out.print("Enter wingspan \t");
+        int wingspan = scanner.nextInt();
+        System.out.print("Enter minimum runway length for take-off \t");
+        int minLengthRunway = scanner.nextInt();
+        System.out.print("Enter count passengers \t");
+        int countPassengers = scanner.nextInt();
+        System.out.print("Enter business class \t");
+        boolean isBusinessClass = scanner.nextBoolean();
+        return civilair = new Civilair(power, top_speed, mass, brand, wingspan, minLengthRunway, countPassengers, isBusinessClass);
```

Comment:
it seems like you don't need a global variable civilair (and others), you can just return a new transport object and that's it